src/pycel/excellib.py

```diff
@@ -844,3 +845,14 @@ def xsum(*args):
 
     # if no non numeric cells, return zero (is what excel does)
     return sum(data)
+
+
+def large(array, k):
+    data = _numerics(array)
+    if isinstance(data, str):
+        return data
+
+    if k == 0 or k > len(data):
```

Comment:
This is probably going to need broader checking.  I suggest you add some test cased for k<0 and for k -> None, bool string, and number string.

Reply:
done, I added a check of k <= 0 and added the `@excel_helper(number_params=1)` decorator which validates k for string, number string or none (and added associated use case in the test)